posts/2020-11-26-MP-rest-client-Kubernetes-secrets-21001.adoc

```diff
@@ -0,0 +1,216 @@
+---
+layout: post
+title: "MP Rest Client 2.0, using Kubernetes secrets in Liberty variables and more Jakarta EE 9 features in Open Liberty 21.0.0.1-beta"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: MP Rest Client 2.0, using Kubernetes secrets in Liberty variables and more Jakarta EE 9 features in Open Liberty 21.0.0.1-beta - OpenLiberty.io
+seo-description: MP Rest Client 2.0, using Kubernetes secrets in Liberty variables. Plus Jakarta Faces Container 3.0 and Jakarta Enterprise Beans Persistent Timer 4.0 included in Jakarta EE 9
+blog_description: MP Rest Client 2.0, using Kubernetes secrets in Liberty variables. Plus Jakarta Faces Container 3.0 and Jakarta Enterprise Beans Persistent Timer 4.0 included in Jakarta EE 9
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= MP Rest Client 2.0, using Kubernetes secrets in Liberty variables and more Jakarta EE 9 features in Open Liberty 21.0.0.1-beta
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+
+Open Liberty 21.0.0.1-beta comes with MP Rest Client 2.0 (which is part of MicoProfile 4.0), this update adds support for HTTP procy servers and automatically following HTTP redirects. In 21.0.0.1-beta you can now use Kubernetes secrets in Liberty variables. Plus Jakarta Faces Container 3.0 and Jakarta Enterprise Beans Persistent Timer 4.0 included in Jakarta EE 9.
```

Comment:
Open Liberty 21.0.0.1-beta provides MP Rest Client 2.0 (which is part of MicoProfile 4.0), which adds support for HTTP proxy servers as well as automatically following HTTP redirects.  Also, you can now use Kubernetes secrets in Liberty variables, plus Jakarta Faces Container 3.0 and Jakarta Enterprise Beans Persistent Timer 4.0 which are part of Jakarta EE 9.

Reply:
Done